### PR TITLE
fix inconsistent util package import clause and some warning message …

### PR DESCRIPTION
**Pull request by bruceauyeung**
What this PR does / why we need it:

this PR makes sure `kubeadmutil.CheckErr()` other than `cmdutil.CheckErr()` is called in `kubeadm version` subcommand. 
in `version.go`, `RunVersion()` function only returns `nil`, `kubeadmutil.CheckErr()` is enough for this

this PR also fix some syntax errors in AlphaWarningOnExit message.

Signed-off-by: redhatlinux10 ouyang.qinhua@zte.com.cn

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/35609)

### CODE REVIEW

File: cmd/kubeadm/app/util/error.go [hunk 1]
```diff
@@ -33,8 +33,8 @@ const (
 )
 
 var AlphaWarningOnExit = dedent.Dedent(`
-	kubeadm: I am an alpha version, my authors welcome your feedback and bug reports
-	kubeadm: please create issue an using https://github.com/kubernetes/kubernetes/issues/new
+	kubeadm: I am at alpha version, my authors welcome your feedback and bug reports
+	kubeadm: please create an issue using https://github.com/kubernetes/kubernetes/issues/new
```
Comment (luxas) [inline cmd/kubeadm/app/util/error.go]: This is already covered in https://github.com/kubernetes/kubernetes/pull/35556

**Comment by luxas** [comment]
LGTM but the rebase confict that will occur

**Comment by redhatlinux10** [comment]
hi @luxas , what should i do for that conflict ?

**Comment by luxas** [comment]
See that #35556 includes that fix already, you should remove it from yours.

**Comment by redhatlinux10** [comment]
@luxas should i wait for that PR being closed ?

**Comment by luxas** [comment]
preferably, yes

Wait until it's merged (which should be very soon)

**Comment by jbeda** [comment]
@k8s-bot ok to test

**Comment by jbeda** [comment]
LGTM -- I'll put the label on after the rebase is pushed.

**Comment by redhatlinux10** [comment]
@jbeda i have pushed the rebase, would you please verify this PR again ?
thanks !

**Comment by luxas** [comment]
Still if would be great if this PR had only one commit, can you please squash?

**Comment by redhatlinux10** [comment]
it seems i failed to squash my commits.  i think i have to abandon this PR and then make a new one, right ?
@jbeda

**Pull request closed without merging**